third_party/terraform/resources/resource_dataflow_flex_template_job.go.erb

```diff
@@ -0,0 +1,187 @@
+<% autogen_exception -%>
+<% unless version == 'ga' -%>
```

Comment:
You'll want to move the `package google` bit up above the version check, Cloud Build is complaining because Go fails to compile packageless files (which the `google` provider one is right now)

Reply:
ahh! that's how it works :)